packages/backend/server/src/app.module.ts

```diff
@@ -39,6 +41,12 @@ export const FunctionalityModules = [
   ScheduleModule.forRoot(),
   EventModule,
   CacheModule,
+  ClsModule.forRoot({
```

Comment:
Provide some description about how to use it? And I didn't see any usage yet, should we import clsmodule right now?

Reply:
its a nestjs binding for [async local storage](https://nodejs.org/api/async_context.html)
that can allow use share some info across the async context, like thread local storage
we can use this context to distinguish multiple async requests that accessing same resource
https://github.com/toeverything/AFFiNE/blob/5c5208bee17ddec642085e033a26706c635cda18/packages/backend/server/src/plugins/redis/mutex.ts#L31-L45

Reply:
I know what it is. If it's not used by now, we should not only import the module